packages/mui-material/src/Tabs/Tabs.test.js

```diff
@@ -712,7 +712,7 @@ describe('<Tabs />', () => {
       tablistContainer.scrollLeft = 0;
       fireEvent.click(findScrollButton(container, 'right'));
       clock.tick(1000);
-      expect(tablistContainer.scrollLeft).equal(220);
+      expect(tablistContainer.scrollLeft).equal(200);
```

Comment:
The problem here is that end-users will never be able to read the 20px, they are always skipped.

Reply:
My reasoning for snapping the scroll to the closest tab bound was to make it a bit easier to follow the current scroll position/index. I do however get that it's not great to skip potentially important content. I can see that this is more important in this case so I'd be fine with this 👍